pkg/api/apis/operators/v1alpha1/subscription_types.go

```diff
@@ -29,13 +29,25 @@ const (
 
 // SubscriptionSpec defines an Application that can be installed
 type SubscriptionSpec struct {
-	CatalogSource          string             `json:"source"`
-	CatalogSourceNamespace string             `json:"sourceNamespace"`
-	Package                string             `json:"name"`
-	Channel                string             `json:"channel,omitempty"`
-	StartingCSV            string             `json:"startingCSV,omitempty"`
-	InstallPlanApproval    Approval           `json:"installPlanApproval,omitempty"`
-	Config                 SubscriptionConfig `json:"config,omitempty"`
+	CatalogSource          string `json:"source"`
+	CatalogSourceNamespace string `json:"sourceNamespace"`
+	// Name is deprecated, prefer Package.
+	Name string `json:"name"`
+	// Package is the name of the package to subscribe to.
+	// +optional
+	Package             string             `json:"package,omitempty"`
+	Channel             string             `json:"channel,omitempty"`
+	StartingCSV         string             `json:"startingCSV,omitempty"`
+	InstallPlanApproval Approval           `json:"installPlanApproval,omitempty"`
+	Config              SubscriptionConfig `json:"config,omitempty"`
+}
+
+func (s *SubscriptionSpec) SpecPackage() string {
```

Comment:
I think `Package()` would make more sense, since this function is on `SubscriptionSpec` already.